src/remotebuild/test/darwinCerts.ts

```diff
@@ -17,7 +17,7 @@
 /* tslint:disable:no-var-requires */
 // var require needed for should module to work correctly
 // Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
-var shouldModule = require("should");
+var shouldModule: any = require("should");
```

Comment:
At one point that resulted in typescript removing the require statement completely because it was "obviously" unused, which is why we went to this approach. They may have fixed their behaviour since then, I'm not sure.